.github/workflows/scala.yml

```diff
@@ -314,7 +314,7 @@ jobs:
       #        uses: andstor/copycat-action@v3
       #        with:
       #          personal_token: ${{ secrets.CI_PAT }}
-      #          src_path: ./distribution/std-lib/docs-js/
+      #          src_path: ./distribution/std-lib/docs-js/ # TODO this path is no longer valid
```

Comment:
make it valid then?

Reply:
XD I guess you are right

For a second I wasn't sure what is the right path here and since it is commented out... But yeah I think I know what the path should be so I'll just change it